content/docs/instrumenting/exposition_formats.md

```diff
@@ -171,6 +158,19 @@ with a tracing system can provide more detailed information related to the speci
 
 To enable this experimental feature you must have at least version v2.26.0 and add `--enable-feature=exemplar-storage` to your arguments.
 
+## Protobuf format
+
+Earlier versions of Prometheus supported an exposition format based on [Protocol Buffers](https://developers.google.com/protocol-buffers/) (aka Protobuf) in addition to the current text-based format. With Prometheus 2.0, the Protobuf format was marked as deprecated and Prometheus stopped ingesting samples from said exposition format.
+
+However, new experimental features were added to Prometheus where the Protobuf format was considered the most viable option. Making Prometheus accept Protocol Buffers once again.
+
+Here is a list of experimental features that, once enabled, will configure Prometheus to favor the Protobug exposition format:
```

Comment:
```suggestion
Here is a list of experimental features that, once enabled, will configure Prometheus to favor the Protobuf exposition format:
```

Reply:
Freudian slip…

Reply:
> Freudian slip

Common, f and g are literally next to each other 🤣